tools/docker/builder/Dockerfile

```diff
@@ -73,6 +73,8 @@ RUN set -xe; \
     tar -xf "${NODEJS_PACKAGE}" -C /opt/; \
     mv "/opt/node-v${NODEJS_VERSION}-${NODEJS_OS}" "${NODEJS_HOME}"; \
     rm -rf "${NODEJS_PACKAGE}"; \
+# Setup Commitlint
+    npm install --location=global @commitlint/cli@17.1.1 \
```

Comment:
I tried running this docker file but it fails with `/bin/sh: 1: npm: not found` can we try installing npm with apt-get

Reply:
I think it would be better to investigate why the nodejs install a few lines above didn't setup npm. Install npm from apt will install a completely separate nodejs.

Reply:
So I noticed few things while looking at this PR
- I approved the PR when I shouldn't have 🤦‍♂️ apologies @elpiel
- NodeJS installs NPM by default but builder image fails to see it as nodejs is added to PATH env only after `RUN` build  is successful https://github.com/build-trust/ockam/blob/d4891db371a8d9daea52f5c0140869b6db4f0385/tools/docker/builder/Dockerfile#L161
We would need to either call `npm` directly from `nodejs directory` or do a `PATH="${NODEJS_HOME}/bin:$PATH"` to set `nodejs` to path.
- `npm install --location=global @commitlint/cli@17.1.1` fails with a [npm ERR! Tracker "idealTree" already exists](https://stackoverflow.com/questions/57534295/npm-err-tracker-idealtree-already-exists-while-creating-the-docker-image-for) error

Reply:
Ok, let's try this again... :joy: 

I've update the PATH env. variable before running `npm` and update the user to `root` when running `npm`.